components/Tournaments/TournamentList.tsx

```diff
@@ -1,34 +1,35 @@
 import { NavigationProp } from '@react-navigation/native';
 import React from 'react'
 import { ViewStyle, StyleSheet } from 'react-native';
+import { TouchableHighlight } from 'react-native-gesture-handler';
 import { Tournament } from '../../models';
 import { FlatList, View, Text, LinkButton } from '../Themed'
 
 interface TournamentListProps {
-    navigation: NavigationProp<any>
-    tournaments: Tournament[]
+    navigation: NavigationProp<any>;
+    tournaments: Tournament[];
+    emptyStateText?: string;
+    onPressTournament: (t: Tournament) => void;
 }
 
-const TournamentList = ({ tournaments, navigation }: TournamentListProps) => {
+const TournamentList = ({ tournaments, navigation, emptyStateText, onPressTournament }: TournamentListProps) => {
 
     const renderItem = ({item}: {item: Tournament}) => {
         return (
-            <View style={styles.row}>
-                <View style={styles.left}>
-                    <Text style={styles.nameText}>{item.name}</Text>
+            <TouchableHighlight style={styles.row} activeOpacity={0.4} underlayColor={'#fff'} onPress={() => onPressTournament(item)}>
+                <View>
```

Comment:
I always forget, `TouchableHighlight` can only have 1 child

Also, styling and onPress styling needs some work, but we can deal with that later